experimental/ComplexReflectionGroups/src/ComplexReflectionGroups.jl

```diff
@@ -0,0 +1,79 @@
+
+# A union type for all algebraic extensions of the field of rational numbers
+QQAlgField = Union{NumField, QQField, QQBarField, QQAbField}
+QQAlgFieldElem = Union{NumFieldElem, QQFieldElem, QQBarFieldElem, QQAbElem}
```

Comment:
These must be `const`, otherwise any code using them is potentially type unstable.
```suggestion
const QQAlgField = Union{NumField, QQField, QQBarField, QQAbField}
const QQAlgFieldElem = Union{NumFieldElem, QQFieldElem, QQBarFieldElem, QQAbElem}
```